bindings/wasm/src/core/fragment.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use wasm_bindgen::prelude::*;
+
+use identity::core::Fragment as Fragment_;
+
+#[wasm_bindgen(js_name = Fragment, inspectable)]
+#[derive(Clone, Hash, PartialEq, Eq, PartialOrd, Ord, Deserialize, Serialize)]
+pub struct WasmFragment(pub(crate) Fragment_);
```

Comment:
`WasmFragment` is another instance where we have typically used `strings` elsewhere in the Wasm bindings for convenience. We should evaluate whether it's worth exporting this at all or just having fallible functions which take `string` and attempt to convert it to the required type.